<launch>
<arg name="dynreconfigure_name" />
<arg name="image_topic" />
<arg name="hz" />
<arg name="scale" />
<arg name="config" />
<arg name="labels" />
<arg name="device" />   
<node pkg="parameter_tuning" type="parameter_tuner.py" name="px4_parameter_tuner" output="screen">
        
	<param name="dynreconfigure_name" type="string" value="$(arg dynreconfigure_name)" />
	<param name="image_topic" type="string" value="$(arg image_topic)" />
	<param name="hz" type="int" value="$(arg hz)" />
	<param name="scale" type="double" value="$(arg scale)" />
	<param name="config" type="string" value="$(arg config)" />
	<param name="labels" type="string" value="$(arg labels)" />
	<param name="device" type="string" value="$(arg device)" />

  </node>
</launch>
